pints/_mcmc/_adaptive_covariance.py

```diff
@@ -115,6 +115,7 @@ def ask(self):
         if not self._running:
             self._running = True
             self._proposed = self._x0
+            self._proposed.setflags(write=False)
```

Comment:
Why do we want to set write to False here?

Reply:
When we accept the point we do `self._current = self._proposed` (no need to create a new array!), and we then want to return the current point as write only

I think we might have allowed users access to `self._proposed` at some point too

Reply:
Cool - want to explain that here? In case someone else later wants to use this as a template to write another MCMC method, they'd know why we want to set this to `write=False`.